src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -517,7 +541,7 @@ def duplicated_mount_point?
       def mounted_devices
         fs_sids = filesystem_devices.map(&:sid)
         devices = Y2Storage::Mountable.all(device_graph)
-        devices = devices.select { |d| !d.mount_point.nil? && !d.mount_point.empty? }
+        devices = devices.select { |d| !d.mount_path.nil? && !d.mount_path.empty? }
```

Comment:
A `mount_point?` method would again come very handy here ;-)

Reply:
Once again, it can be replaced by `!d.mount_point.nil?`